.github/workflows/ci.yaml

```diff
@@ -130,11 +130,65 @@ jobs:
           name: coverage-${{ matrix.python-version }}
           path: .coverage
 
+  tests-linux-alpha:
+    name: tests / run / ${{ matrix.python-version }} / Linux
+    runs-on: ubuntu-latest
+    timeout-minutes: 20
+    strategy:
+      matrix:
+        python-version: ["3.11-dev"]
+    outputs:
+      python-key: ${{ steps.generate-python-key.outputs.key }}
```

Comment:
The job `output` isn't actually used anywhere. So this is safe to remove.
`${{ steps.python.outputs.python-version }}` does work without it.